CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [2.5.0] - 2017-07-06
```

Comment:
please remove any versioning as we do this post acceptance. We do this for several reasons:
- PRs are not FIFO, by versioning before acceptance can lead to the submitter needing to rebase more often
- maintainers may disagree with the impact of the issue

Also in this case `2.6.0`  has already been released so this would be `2.7.0` but leave versioning to maintainers anyways is a better option.